communication/meeting-notes-archive/201911-202010_Community_Meeting_Notes.md

```diff
@@ -961,7 +961,7 @@ I want to call out [@karenb](https://kubernetes.slack.com/team/UCLQ9GKSP) dealin
                 *   People start to depend on non-ga beta things
                 *   Timebox amount of time a feature is allowed to stay in beta.
             *   [Plan to set up CI to ensure conformance coverage](https://github.com/kubernetes/enhancements/pull/1306)
-            *   [Plan to eliminate dependencies on non-GA features in conformance](http://git.k8s.io/enhancements/keps/sig-architecture/20191023-conformance-without-beta.md)
+            *   [Plan to eliminate dependencies on non-GA features in conformance](https://git.k8s.io/enhancements/keps/sig-architecture/20191023-conformance-without-beta.md)
```

Comment:
404 error...

Reply:
```suggestion
            *   [Plan to eliminate dependencies on non-GA features in conformance](https://git.k8s.io/enhancements/keps/sig-architecture/1333-conformance-without-beta/README.md)
```
changed as part of https://github.com/kubernetes/enhancements/commit/19a49e146f583850a783fa2960906626468cdb70